tslearn/preprocessing.py

```diff
@@ -131,10 +130,12 @@ class TimeSeriesScalerMeanVariance(TransformerMixin):
     def __init__(self, mu=0., std=1.):
         self.mu_ = mu
         self.std_ = std
+        self.global_mean = None
+        self.global_std = None
 
-    def fit_transform(self, X, **kwargs):
+    def fit_transform(self, X, global_stats=False, **kwargs):
```

Comment:
Hum, `global_stats` argument is never used afterwards, is it?

If not, you should remove it, I guess.

Reply:
i think we actually made a reasoning mistake and thought the normalization happened dataset-wise, I think the global_stats and `self.global_mean` + `self.global_std` stem from that. Of course, this becomes unnecessary since the normalization happens timeseries-wise. We just have to use the `transform` method of the SAME normalizer on both the provided X input array as the centroids. 

Important to note in the documentation is that the centroids may not be normalized by the user himself.